lib/interpreters/compute_several_solutions.rb

```diff
@@ -216,6 +216,8 @@ def self.find_best_heuristic(service_vrp)
 
             Models::Route.new(vehicle: vrp.vehicles.find{ |v| v[:id] == route[:vehicle_id] }, mission_ids: mission_ids)
           }.compact
+        else
+          best[:heuristic] = nil
```

Comment:
If we do it like this, the `synthesis` output will not show that the supplied routes are used as first solution. I suggest implementing the logic directly at the line below without going through `nil`.